test/runtests.jl

```diff
@@ -0,0 +1,17 @@
+using ChainRulesCore
+using ChainRulesTestUtils
+using Random
+using Test
+
+@testset "ChainRulesTestUtils.jl" begin
+    double(x) = 2x
+    @scalar_rule(double(x), 2)
+    test_scalar(double, 2)
+
+    fst(x, y) = x
+    frule(::typeof(fst), x, y, _, dx, dy) = (x, dx)
+    rrule(::typeof(fst), x, y)=  fst_pullback(dout) = (NO_FIELDS, @thunk(identity(dout)), Zero())
+
+    frule_test(fst, (2, 4))
```

Comment:
```suggestion
    frule_test(fst, (2, 4.0), (3, 5.0))
```
`fst` is a binary function, so needs two tuple of arguments :)

Reply:
With these changes the test still fails on the same issue.

```
ChainRulesTestUtils.jl: Error During Test at /Users/mattbr/.julia/dev/ChainRulesTestUtils.jl/test/runtests.jl:6
  Got exception outside of a @test
  MethodError: no method matching iterate(::Nothing)
  Closest candidates are:
    iterate(!Matched::Core.SimpleVector) at essentials.jl:600
    iterate(!Matched::Core.SimpleVector, !Matched::Any) at essentials.jl:600
    iterate(!Matched::ExponentialBackOff) at error.jl:218
    ...

```